### ISSUE TITLE
Cannot read property '0' of undefined [on EuPMC]

### ISSUE THREAD

**Issue by petermr** [ContentMine/getpapers#67]
```
localhost:zika pm286$ getpapers --version
0.4.1
```
```
getpapers --query "Zika" --outdir zika --all --pdf --xml
...
end of output...

warn: Article with title "Effect of salts and sugars on the clarity of gellan gels did not have a PMCID (therefore no XML)
info: Got XML URLs for 138 out of 550 results
info: Downloading fulltext XML files
Downloading files [=====================] 100% (138/138) [3.4s elapsed, eta 0.0]
info: All XML downloads succeeded!
warn: Article with pmid "26820387" had no fulltext PDF url
warn: Article with pmid "26796813" had no fulltext PDF url
warn: Article with pmid "26820244" had no fulltext PDF url
warn: Article with pmid "26820163" had no fulltext PDF url
warn: Article with pmid "26794427" had no fulltext PDF url
warn: Article with pmid "26782128" had no fulltext PDF url

/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/lib/eupmc.js:464
  var urls = result.fullTextUrlList[0].fullTextUrl;
                                   ^
TypeError: Cannot read property '0' of undefined
    at EuPmc.getFulltextPDFUrl (/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/lib/eupmc.js:464:36)
    at Array.map (native)
    at EuPmc.downloadFulltextPDFs (/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/lib/eupmc.js:250:6)
    at EuPmc.nextDlTask (/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/lib/eupmc.js:199:5)
    at /Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/lib/eupmc.js:240:11
    at /Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/node_modules/lodash/index.js:7305:23
    at Object.oncomplete (fs.js:108:15)
```

**Comment by Blahah** [comment]
What version have you got peter? Please make sure it's the latest one.

**Comment by Blahah** [comment]
Ah, sorry I see you included the version. You don't have the latest but the bug fixed in the latest is not this one.

**Comment by tarrow** [comment]
This may well now be fixed by pull request #72

**Comment by Blahah** [comment]
Indeed, fixed in #72 and #77
